READEM2.MD

```diff
@@ -0,0 +1 @@
+dummy file for creating a new empty branch for Staging Spoke instance of MoveOn/Spoke latest branch main
```

Comment:
please remove?  or what's this for?

Reply:
The WFP folks added this file.  I'll remove it.

Reply:
removed

Reply:
removed -- it got in here because I merged WFP's staging into my branch